src/interfaces/store-interfaces/state-interfaces/registration-state-interface.ts

```diff
@@ -15,4 +15,5 @@ export interface RegistrationStateIF {
   naics: NaicsIF
   businessType: BusinessTypes
   businessNumber?: string // GPs only
+  BusinessTypeConfirm?: boolean // GPs only
```

Comment:
Our code style for variables is camelCase.

We use PascalCase for Types, Enums an const classes.